octodns_gcore/__init__.py

```diff
@@ -594,6 +708,7 @@ def _apply_update(self, change):
         self.log.info("updating: %s", change)
         new = change.new
         data = getattr(self, f"_params_for_{new._type}")(new)
+        self.log.debug("updating: %s", str(data))
```

Comment:
This looks like a test/debugging log message rather than something targeted at/useful for end users. Intentionally left in?

Reply:
Yes. It's a debugging log. It was helpful during the dev but can be safely removed.